### DISCUSSION TITLE
Loop through multiple collections

### DISCUSSION THREAD

**Discussion by nicktc** [statamic/cms#4781] [Q&A] [answered]
I have created three collections:

- Vacancies
- News
- Customer cases

On most pages I only show the vacancies, or the customer cases. But I have this one page ('Updates') where I would like to show all entries from these 3 different collections, nicely sorted on date.

Is something like this possible, or would that mean that I would have to put all of these three things into 1 collection?

**Comment by jasonvarga** [comment, marked as answer]
You can pass multiple handles into the collection tag.

```
{{ collection from="vacancies|news|customer_cases" }}
```

https://statamic.dev/tags/collection

**Comment by nicktc** [comment]
Exactly what I need!
